.github/workflows/pfx_to_file.yml

```diff
@@ -1,22 +1,42 @@
 # Workflow to test secrets, first version empty in order to be able to run it from branch
-
 name: PFX To File
 
-# Controls when the workflow will run
 on:
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
-
-    # Steps represent a sequence of tasks that will be executed as part of the job
-    steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v3
-      - name: Write key file
-          ls -la
+    build:
+        runs-on: ubuntu-latest
```

Comment:
Could we set a specific Ubuntu version?

Reply:
Cause there's no dependency on the platform I would like to use the latest